src/main/java/frc/robot/subsystems/Swerve/DrivebaseSubsystem.java

```diff
@@ -184,15 +190,29 @@ public Rotation2d getGyroscopeRotation() {
     public void periodic() {
 
         swerveOdometry.update(getGyroscopeRotation(), getModulePositions());
-
+        System.out.println(ll.getRobotPose());
         Optional<EstimatedRobotPose> result = pcw.getEstimatedGlobalPose(swerveOdometry.getEstimatedPosition());
-        /* 
+
         if (result.isPresent()) {
             EstimatedRobotPose camPose = result.get();
             swerveOdometry.addVisionMeasurement(
                     camPose.estimatedPose.toPose2d(),
                     camPose.timestampSeconds);
-        }*/
+        }
+        LimelightResults llresults = LimelightHelpers.getLatestResults("limelight");
+
+        NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
```

Comment:
It feels weird to read a ton of data from the Limelight as JSON, and materialize a complex structure object from it, only to re-read three more variables from the device directly. I do not know if your second reads are from cache and consistent with the first read, or if you have inconsistent values here. If it's working now, roll with it for GPK, but it's a question you surely want to understand.